Source/JavaScriptCore/runtime/JSBigInt.cpp

```diff
@@ -440,6 +440,10 @@ class HeapBigIntImpl {
     JSBigInt* m_bigInt;
 };
 
+// For use in places where we could negate std::numeric_limits<T>::min and would like to avoid UB.
+template<std::integral T>
+ALWAYS_INLINE T negate(T v) { return ~static_cast<std::make_unsigned_t<T>>(v) + 1; }
```

Comment:
Oh I would have put this in `WTF/MathExtras.h`

Reply:
Thanks. Yes, I missed that part.